tests/pkg/utils/mco_managedcluster.go

```diff
@@ -71,3 +72,85 @@ func ListManagedClusters(opt TestOptions) ([]string, error) {
 
 	return clusterNames, nil
 }
+
+func ListOCPManagedClusterIDs(opt TestOptions, minVersionStr string) ([]string, error) {
+	minVersion, err := goversion.NewVersion(minVersionStr)
+	if err != nil {
+		return nil, err
+	}
+	clientDynamic := GetKubeClientDynamic(opt, true)
+	objs, err := clientDynamic.Resource(NewOCMManagedClustersGVR()).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	clusterIDs := []string{}
+	for _, obj := range objs.Items {
+		metadata := obj.Object["metadata"].(map[string]interface{})
+		labels := metadata["labels"].(map[string]interface{})
+		if labels != nil {
+			vendorStr := ""
+			if vendor, ok := labels["vendor"]; ok {
+				vendorStr = vendor.(string)
+			}
+			obsControllerStr := ""
+			if obsController, ok := labels["feature.open-cluster-management.io/addon-observability-controller"]; ok {
+				obsControllerStr = obsController.(string)
+			}
+			if vendorStr == "OpenShift" && obsControllerStr == "available" {
+				clusterVersionStr := ""
+				if clusterVersionVal, ok := labels["openshiftVersion"]; ok {
+					clusterVersionStr = clusterVersionVal.(string)
+				}
+				clusterVersion, err := goversion.NewVersion(clusterVersionStr)
+				if err != nil {
+					return nil, err
+				}
+				if clusterVersion.GreaterThanOrEqual(minVersion) {
+					clusterIDStr := ""
+					if clusterID, ok := labels["clusterID"]; ok {
+						clusterIDStr = clusterID.(string)
+					}
+					if len(clusterIDStr) > 0 {
+						clusterIDs = append(clusterIDs, clusterIDStr)
+					}
+				}
+			}
+		}
+	}
+
+	return clusterIDs, nil
+}
+
+func ListKSManagedClusterNames(opt TestOptions) ([]string, error) {
+	clientDynamic := GetKubeClientDynamic(opt, true)
+	objs, err := clientDynamic.Resource(NewOCMManagedClustersGVR()).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	clusterNames := []string{}
+	for _, obj := range objs.Items {
+		metadata := obj.Object["metadata"].(map[string]interface{})
+		labels := metadata["labels"].(map[string]interface{})
+		if labels != nil {
+			vendorStr := ""
+			if vendor, ok := labels["vendor"]; ok {
+				vendorStr = vendor.(string)
+			}
+			obsControllerStr := ""
+			if obsController, ok := labels["feature.open-cluster-management.io/addon-observability-controller"]; ok {
+				obsControllerStr = obsController.(string)
+			}
+			if (vendorStr == "GKE" || vendorStr == "EKS" || vendorStr == "AKS") && obsControllerStr == "available" {
```

Comment:
we support all *KS. so here suggest to use `vendorStr != `OpenShift`

Reply:
done